### ISSUE TITLE
groups: groups sidebar won't load on Safari

### ISSUE THREAD

**Issue by bonbud-macryg** [tloncorp/landscape-apps#1400]
**Describe the bug, optionally with screenshots**
In Safari v16.1 (running macOS Ventura) I can access individual groups by clicking notifications from Grid, but when I click the Groups tile icon the sidebar doesn't load. ([Screenshot](https://0x0.st/odJ3.png))

I get the same behaviour on Escape's groups sidebar. ([Screenshot](https://0x0.st/odyB.png))

The sidebar loads as expected on Chrome and Firefox.

EDIT: Clicking through to Groups from Grid works fine on my moon, accessed on the same browser and running on the same machine. The big difference is that my moon is only in the groups Rosetta Code and Urbit Community.

Javascript console output after opening Groups 2 from the tile icon in Safari:
```
[Warning] The whitelist option is deprecated and will be removed in the next version. Please use the 'partialize' option instead. (index.d96bf496.js, line 115)
[Warning] The whitelist option is deprecated and will be removed in the next version. Please use the 'partialize' option instead. (index.d96bf496.js, line 115)
[Warning] The whitelist option is deprecated and will be removed in the next version. Please use the 'partialize' option instead. (index.d96bf496.js, line 115)
[Warning] The whitelist option is deprecated and will be removed in the next version. Please use the 'partialize' option instead. (index.d96bf496.js, line 115)
[Error] Unhandled Promise Rejection: QuotaExceededError: The quota has been exceeded.
d (index.d96bf496.js:115:7020)
(anonymous function) (index.d96bf496.js:115:7103)
(anonymous function) (index.d96bf496.js:116:19458)
asyncFunctionResume
(anonymous function)
promiseReactionJobWithoutPromise
promiseReactionJob
[Error] Failed to load resource: You do not have permission to access the requested resource.
```

**Additional context:**
Self-hosted planet running on a MacBook Pro, accessed via Tailscale.

**Comment by rcrdlbl** [comment]
I can't repro this on safari 16.0 – it does take a while to load for me but not noticeably longer than on chrome

**Comment by bonbud-macryg** [comment]
* HTTP
* I've cleared Safari's cache via both methods described [here](https://www.macrumors.com/how-to/clear-safari-cache/) and refreshed after each attempt, and see no change.

**Comment by rcrdlbl** [comment]
recent versions of safari seem to be "always-on HTTPS if a certificate is available" and therefore I can't easily replicate this with a ship running a lot of groups, but i can't replicate this using HTTP on a fakezod running locally

**Comment by rcrdlbl** [comment]
the `QuotaExceededError` seems to be related to this though – your ship may be maxxing out the LocalStorage that safari affords users

**Comment by bonbud-macryg** [comment]
Worth noting that I also have this problem with Safari on iOS 16.0.

**Comment by rcrdlbl** [comment]
okay, this description makes it sound pretty likely this is related to #1397, and it starts breaking things on safari because it sets a lower maximum for the amount of things you can put in LocalStorage, but we'll have to run further testing

@bonbud-macryg if you can get this ship accessible on HTTPS, i'd see if you run into the same issue, that can help us narrow this down quite a bit, but otherwise just stay posted.

**Comment by rcrdlbl** [comment]
oh, i forgot one more important environment question, @bonbud-macryg:

how many groups are you in at the moment?

**Comment by bonbud-macryg** [comment]
* 84 groups, of which 20 have still to migrate.
* I've had "set up HTTPS" on the to-do list for ages, I think I messed it up the first time I tried so it would be a bit of a project to fix it all up and retry.

**Comment by jamesacklin** [comment]
May be DDOSing your ship with getting group info all at once, first mitigating fix is in #1434
